tests/Integration/Integrations/FacebookInstantArticlesTest.php

```diff
@@ -17,35 +18,63 @@
  * Test Facebook Instant Articles integration.
  */
 final class FacebookInstantArticlesTest extends TestCase {
-	// This values should mirror the values in `class-facebook-instant-articles.php`.
-	private const REGISTRY_IDENTIFIER   = 'parsely-analytics-for-wordpress';
-	private const REGISTRY_DISPLAY_NAME = 'Parse.ly Analytics';
+	/**
+	 * Internal variable.
+	 *
+	 * @var Facebook_Instant_Articles $fbia Holds the Facebook_Instant_Articles object.
+	 */
+	private static $fbia;
+
+	/**
+	 * Internal variable.
+	 *
+	 * @var string $registry_identifier Hols the same value as the private constant in the class.
+	 */
+	private static $registry_identifier;
+
+	/**
+	 * Internal variable.
+	 *
+	 * @var string $registry_display_name Hols the same value as the private constant in the class.
+	 */
+	private static $registry_display_name;
+
+	/**
+	 * The setUp run before each test
+	 */
+	public function set_up(): void {
+		parent::set_up();
+
+		self::$fbia = new Facebook_Instant_Articles();
```

Comment:
Be careful with instantiating the class under test in a set_up() method. If the class under test has properties that are set in one test, then it can inadvertently effect the behaviour of other tests.